src/rules/function-url-quotes/__tests__/index.js

```diff
@@ -145,6 +145,10 @@ testRule(rule, {
     code: "@font-face { font-family: 'foo'; src: url($variable + \"foo.ttf\" + $variable); }",
   }, {
     code: "a { background: url(\"/images/my_image@2x.png\") }",
+  }, {
+    code: "@-moz-document url-prefix() {}",
+  }, {
+    code: "a { background: url() }",
```

Comment:
@evilebottnawi Why? I think this rule check that url() argument has quotes. And `""` has quotes.